crates/mongodb-support/src/bson_type.rs

```diff
@@ -259,7 +278,60 @@ impl BsonScalarType {
             S::Undefined => true,
             S::DbPointer => true,
             S::Symbol => true,
+            S::UUID => true,
+        }
+    }
+
+    /// True iff we consider a to be a supertype of b.
```

Comment:
Nit typo

Reply:
That's not a typo - "iff" is short for "if and only if"